packages/google_maps_flutter/google_maps_flutter_platform_interface/README.md

```diff
@@ -0,0 +1,26 @@
+# google_maps_flutter_platform_interface
+
+A common platform interface for the [`google_maps_flutter`][1] plugin.
```

Comment:
This is a link but instead of hardcoding the URL inline, you can refer to 1 at the bottom of the document (it can be anything, really).

So you can have something like:

```
This is my [link][1] blah blah blah

...

[1]: https://www.google.com
```

That way you can keep your links all together at the bottom, instead of inlined into your documentation.

(See Reference-style links [here](https://daringfireball.net/projects/markdown/syntax#link))